salt/returners/etcd_return.py

```diff
@@ -69,11 +93,12 @@ def __virtual__():
     return __virtualname__ if HAS_LIBS else False
 
 
-def _get_conn(opts):
+def _get_conn(opts, profile=None):
     '''
     Establish a connection to etcd
     '''
-    profile = opts.get('etcd.returner', None)
+    if profile is None:
+        profile = opts.get('etcd.returner', None)
```

Comment:
The `None` is redundant here, FWIW. It's the default from the `get()` for a failed lookup.